recipes/intarna/meta.yaml

```diff
@@ -1,28 +1,32 @@
 package:
   name: intarna
-  version: "1.2.5"
+  version: 2.0.0
 
 about:
   home: http://www.bioinf.uni-freiburg.de/Software
-  license: GPL
-  license_file: COPYING
-  summary: Efficient target prediction incorporating accessibility of interaction sites
+  license: MIT
+  license_file: LICENSE
```

Comment:
Martin, if you don't have a license_file that is called LICENSE that it will break. You can also, and maybe should, add such a license file to your repo.